app/src/features/surveys/components/StudyAreaForm.tsx

```diff
@@ -5,25 +5,37 @@ import { useFormikContext } from 'formik';
 import { Feature } from 'geojson';
 import yup from 'utils/YupSchema';
 
-export interface IStudyAreaForm {
-  location: {
-    survey_area_name: string;
-    geometry: Feature[];
-  };
+export interface ISurveyLocationForm {
+  locations: {
+    survey_location_id?: number;
+    name: string;
+    description: string;
+    geojson: Feature[];
+    revision_count?: number;
+  }[];
 }
 
-export const StudyAreaInitialValues: IStudyAreaForm = {
-  location: {
-    survey_area_name: '',
-    geometry: []
-  }
+export const SurveyLocationInitialValues: ISurveyLocationForm = {
+  locations: [
+    {
+      survey_location_id: null as unknown as number,
+      name: '',
+      // TODO description is temporarily hardcoded until the new UI to populate this field is implemented
```

Comment:
If there is an existing ticket to address this, it could be a good idea to link it

Reply:
Yup, I believe its this one in the current sprint..? https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-219

will add